CONTRIBUTING.md

```diff
@@ -42,7 +42,7 @@ Notice: The easiest way to get started is to copy an existing language class clo
 
 Add your new templates as listed in your language class **getFiles** method. Make sure to follow the [checklist](https://github.com/appwrite/sdk-generator#sdk-checklist) when building the language templates.
 
-Make sure to follow the objects structure and service separation architecture. We aim to keep developer experience as consisted as possible across different SDKs to make the learning curve as small as possible.
+Make sure to follow the objects structure and service separation architecture. We aim to keep developer experience as consistant as possible across different SDKs to make the learning curve as small as possible.
```

Comment:
`consistant` -> `consistent`

Reply:
Nice catch! Shall update this shortly and commit